app/bundles/EmailBundle/Form/Type/DashboardMostHitEmailRedirectsWidgetType.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Mautic\EmailBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Class DashboardMostHitEmailRedirectsWidgetType.
+ */
+class DashboardMostHitEmailRedirectsWidgetType extends AbstractType
+{
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array                $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add(
+            'companyId',
+            'company_list',
+            [
+                'label'       => 'mautic.email.companyId.filter',
+                'label_attr'  => ['class' => 'control-label'],
+                'attr'        => ['class' => 'form-control'],
+                'empty_data'  => '',
+                'required'    => false,
+                'multiple'    => false,
+                'modal_route' => null, // disable "Add new" option in ajax lookup
+            ]
+        );
+
+        $builder->add(
+            'campaignId',
+            'campaign_list',
+            [
+                'label'      => 'mautic.email.campaignId.filter',
+                'label_attr' => ['class' => 'control-label'],
+                'attr'       => ['class' => 'form-control'],
+                'empty_data' => '',
+                'required'   => false,
+                'multiple'   => false,
+            ]
+        );
+
+        $builder->add(
+            'segmentId',
+            'leadlist_choices',
+            [
+                'label'      => 'mautic.email.segmentId.filter',
+                'label_attr' => ['class' => 'control-label'],
+                'attr'       => ['class' => 'form-control'],
+                'empty_data' => '',
+                'required'   => false,
+            ]
+        );
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
```

Comment:
Oh and I just remembered that Petr commented on one of my PRs that this method is deprecated. Use the form type classname instead. https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.8.md#form